### PR TITLE
Renewing docs/user-guide.md

### LINKED ISSUE #59: The shell in user-guide should be changed.
**What happened**:
In docs/user-guide.md, the shell about "run the following command to load images into the local docker space" gets a repetitive operation. It might confuse the regular users. 

**What you expected to happen**:

`cp hack/images/image.list.load.online.example cp hack/images/image.list.load.online
bash hack/ops/load-images.sh hack/images/image.list.load.online` 
should be changed by 
`cp hack/images/image.list.load.online.example
bash hack/ops/load-images.sh hack/images/image.list.load.online` 
or better solution.

### PR DESCRIPTION
**Pull request by chouchongYHMing**
**What type of PR is this?**

/kind documentation

**Which issue(s) this PR fixes**:
Fixes #59 

**Special notes for your reviewer**:
@warjiang 
**Does this PR introduce a user-facing change?**:
I change the shell to 
```
cp hack/images/image.list.load.online.example  hack/images/image.list.load.online
bash hack/ops/load-images.sh hack/images/image.list.load.online
```

### CODE REVIEW

**Comment by RainbowMango** [comment]
@chouchongYHMing Thanks for doing this.
You might have to sign the DCO as per [the guideline](https://github.com/karmada-io/dashboard/pull/60/checks?check_run_id=27595518717) to make CI happy.

/assign @warjiang

File: hack/images/image.list.load.online
Comment (warjiang) [inline hack/images/image.list.load.online:1]: the `hack/images/image.list.load.online` should not be commited, it depends on the scenarios

**Comment by chouchongYHMing** [comment]
@RainbowMango I have solved the problem with DCO.

File: hack/images/image.list.load.online
Comment (chouchongYHMing) [reply hack/images/image.list.load.online:1]: I have taken the advice.

Comment (RainbowMango) [reply hack/images/image.list.load.online:1]: Then, we have to get rid of this file, right?

Comment (warjiang) [reply hack/images/image.list.load.online:1]: Yes~

File: hack/images/image.list.load.online.example [hunk 1]
```diff
@@ -26,4 +26,4 @@ karmada-scheduler-estimator;docker.io/karmada/karmada-scheduler-estimator:v1.9.0
 karmada-aggregated-apiserver;docker.io/karmada/karmada-aggregated-apiserver:v1.9.0;;;
 karmada-search;docker.io/karmada/karmada-search:v1.9.0;;;
 karmada-metrics-adapter;docker.io/karmada/karmada-metrics-adapter:v1.9.0;;;
-karmada-agent;docker.io/karmada/karmada-agent:v1.9.0;;;
+karmada-agent;docker.io/karmada/karmada-agent:v1.9.0;;;
```
Comment (RainbowMango) [inline hack/images/image.list.load.online.example:29]: No end of the line. not supposed to change.
But no big deal :)

**Review by RainbowMango — COMMENTED** [review]
Generally looks good to me.
Would you mind [squash commits](https://karmada.io/docs/next/contributor/github-workflow/#squash-commits)?

File: hack/images/image.list.load.online.example (hunk 1 above)
Comment (chouchongYHMing) [reply hack/images/image.list.load.online.example]: I am trying to squash commits.🙂

**Comment by warjiang** [comment]
/remove-tide/merge-method-squash

**Pull request merged**